src/Umbraco.Core/Services/IMemberService.cs

```diff
@@ -219,6 +219,18 @@ IMember CreateMemberWithIdentity(string username, string email, string name, str
     /// </returns>
     IEnumerable<IMember> GetMembersByMemberType(string memberTypeAlias);
 
+    /// <summary>
+    ///     Get an list of <see cref="IMember"/> for all members with the specified email.
+    /// </summary>
+    //// <param name="email">Email to use for retrieval</param>
+    /// <returns>
+    ///     <see cref="IEnumerable{IMember}" />
+    /// </returns>
+    IEnumerable<IMember> GetMembersByEmail(string email)
+    {
+        return Enumerable.Empty<IMember>();
+    }
```

Comment:
Adding default implementations to interfaces to avoid breaking changes is a great solution, but it should still return a usable result or (worst case) an exception indicating the method isn't implemented in the inherited class...

In this case, it could lookup all members and filter in-memory (which is really inefficient, but still return the expected value), only return a single member by using the `GetByEmail()` method or actually use the paged `FindByEmail()` method using `StringPropertyMatchType.Exact`:

```suggestion
    /// <summary>
    ///     Get an list of <see cref="IMember"/> for all members with the specified email.
    /// </summary>
    /// <param name="email">Email to use for retrieval</param>
    /// <returns>
    ///     <see cref="IEnumerable{IMember}" />
    /// </returns>
    IEnumerable<IMember> GetMembersByEmail(string email)
        => FindByEmail(email, 0, int.MaxValue, out _, StringPropertyMatchType.Exact); // TODO: Remove default implementation in v14
```

Although this means there's already an existing method implemented that allows you to get all members by email, it does have the overhead of pagination and ordering by the email column... And having an easy to use `GetMembersByEmail()` method still adds value, so I'd keep this new method 👍🏻